src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -10,9 +10,17 @@
 //import edu.wpi.first.wpilibj.networktables.NetworkTable;
 import edu.wpi.cscore.UsbCamera;
 import edu.wpi.first.wpilibj.CameraServer;
-import edu.wpi.first.wpilibj.TimedRobot;
+
+import edu.wpi.first.wpilibj.IterativeRobot;
+import edu.wpi.first.wpilibj.networktables.NetworkTable;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
+import org.usfirst.frc.team467.robot.Autonomous.ActionGroup;
+// import org.usfirst.frc.team467.robot.Autonomous.Actions;
```

Comment:
delete this commented Autonomous actions import if you import it the very next line.